src/Octoliner.h

```diff
@@ -1,102 +1,48 @@
 /*
- * This file is a part of Dragster car set library.
+ * This file is a part of Octoliner library.
  *
- * Defines: octoliner object - 8-point line-folowing sensor driver
- * � Amperka LLC (https://amperka.com, dev@amperka.com)
+ * Product page: https://amperka.ru/product/zelo-folow-line-sensor
+ * © Amperka LLC (https://amperka.com, dev@amperka.com)
  * 
  * Author: Vasily Basalaev <vasily@amperka.ru>
  * Refactored by: Yury Botov <by@amperka.com>
  * License: GPLv3, all text here must be included in any redistribution.
  */
 
+#include "GpioExpander.h"
 #include <Arduino.h>
+#include <Wire.h>
 
 #ifndef __OCTOLINER_H__
 #define __OCTOLINER_H__
 
-#ifndef INPUT_PULLDOWN
-#define INPUT_PULLDOWN 0x3
-#endif
-//INPUT_PULLDOWN
-
-#ifndef DEFAULT_GPIOEXP_ADDR
-#define DEFAULT_GPIOEXP_ADDR 42
-#endif
-// DEFAULT_GPIOEXP_ADDR
+class Octoliner : public GpioExpander {
```

Comment:
```suggestion
class Octoliner : private GpioExpander {
```

We don’t want to expose the internal details of `GpioExpander`. Don’t forget to expose `setAddress` and `saveAddress` though.

Reply:
...and add public definitions for change and saveAddress. Done.